torchvision/utils.py

```diff
@@ -30,16 +28,15 @@ def make_grid(
         padding (int, optional): amount of padding. Default: ``2``.
         normalize (bool, optional): If True, shift the image to the range (0, 1),
             by the min and max values specified by :attr:`range`. Default: ``False``.
-        range (tuple, optional): tuple (min, max) where min and max are numbers,
-            then these numbers are used to normalize the image. By default, min and max
-            are computed from the tensor.
+        value_range (tuple, optional): Attention: range changed to value_rangetuple.
```

Comment:
If you add the `**kwargs` and throw a warning, I think you can remove the `Attention: range changed to value_rangetuple.` from here.